Source/Core/Core/PowerPC/Interpreter/Interpreter_Branch.cpp

```diff
@@ -2,12 +2,12 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "Core/PowerPC/Interpreter/Interpreter.h"
```

Comment:
Apparently `IncludeIsMainRegex` should work for that, but it is set to `"$"` (ie. do not allow any suffix) at this point. What about making it `"(_.+)?$"` (ie. underscore followed by anything else)?